jobs/workflow_test_release.groovy

```diff
@@ -66,16 +66,21 @@ job(name) {
     colorizeOutput 'xterm'
     credentialsBinding {
       string("GCLOUD_CREDENTIALS", "246d6550-569b-4925-8cda-e11a4f0d6803")
+      string("AUTH_TOKEN", "a62d7fe9-5b74-47e3-9aa5-2458ba32da52")
     }
   }
 
   steps {
     shell """
       #!/usr/bin/env bash
-
       set -eo pipefail
-
-      WORKFLOW_CHART="workflow-\${RELEASE}" WORKFLOW_E2E_CHART="workflow-\${RELEASE}-e2e" ./ci.sh
+      export WORKFLOW_CHART="workflow-\${RELEASE}"
+      export WORKFLOW_E2E_CHART="workflow-\${RELEASE}-e2e"
+      env > /tmp/\${JOB_NAME}/\${BUILD_NUMBER}/env.file
```

Comment:
since both `workflow-test(-pr)` and `workflow-test-release` jobs appear to run the same logic contained on these lines 79-83, wondering if we can extract it into either a common/re-usable string in `common.groovy` and/or a similarly accessible bash script that multiple jobs can reference?